docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebar_label: Try GX Cloud
```

Comment:
Call it getting started with GX Cloud

Reply:
@joshzzheng I think Try GX Cloud is better. If you recall, we are having a LOT of issues with the _[Quickstart](https://docs.greatexpectations.io/docs/tutorials/quickstart/)_ for OSS. Users are often confused when they discover that the set up and configuration of GX is not quick or simple and a completely different process. The same can be said for GX Cloud - the process of connecting to a Data Source is complex and isn't straightforward. The current topic title is accurate and I recommend that we keep it